packages/vscode-extensions/src/browser/vscode-extensions-api.ts

```diff
@@ -0,0 +1,79 @@
+/********************************************************************************
```

Comment:
File name and api name is confusing, I believe I does not have anything to do with vscode extension apis, but with registry api